app/views/local_authorities/index.html.erb

```diff
@@ -19,14 +19,13 @@
           <div class='count'><span><%= @authorities.count %></span> local authorities</div>
         </th>
       </tr>
-      <%= render partial: "govuk_admin_template/table_filter",
-      locals: {placeholder: "Filter list of local authorities"} %>
     </thead>
     <tbody>
       <% @authorities.each do |authority| %>
         <tr>
-          <td class="name">
-            <%= link_to authority.name, local_authority_services_path(authority.slug), class: 'js-open-on-submit' %>
+          <td>
+            <%= link_to authority.name, local_authority_services_path(authority.slug), class: 'js-open-on-submit local-authority-name' %>
+            <div class="count"><span><%= authority.broken_link_count %> </span>Broken links</div>
```

Comment:
Could we add a test for this to the integration specs? Not sure how fiddly testing the rows would be...